analyses/immune-deconv/02-visualize_quantiseq.Rmd

```diff
@@ -376,8 +407,7 @@ ratio_plot <- ggplot(ratio_df) +
       y = score) +
   # remove outliers
   geom_boxplot(outlier.shape = NA, color = "grey40", size = 0.2) + 
-  geom_jitter(width = 0.15, size = 0.75, alpha = 0.8, aes(color = cancer_group_hex)) + 
-  #facet_wrap(~cancer_group_display, nrow = 2, scales = "free") +
+  geom_jitter(shape = 1, width = 0.15, size = 0.75, alpha = 0.6) + 
```

Comment:
Since this plot appears next to the one we added the legend too, I think we can use the same cancer group colors here (`color = cancer_group_hex`), and revert back to default shapes (no `shape=1` argument). We can indicate in the caption that colors are same as in adjacent panel.